### PR TITLE
Render json strings in popover of attributes table

### LINKED ISSUE #2801: Render json strings in the service instance attributes
Because of limitations in the API we store JSON in some attributes. These are shown as strings, however this is not very readable for users.

The goal of this ticket is to render a value as json when it is valid json. Do a stringify

### PR DESCRIPTION
**Pull request by andraskvr**
# Description

closes #2801 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [x] ~End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx [hunk 1]
```diff
@@ -32,12 +32,15 @@ export const CellWithCopy: React.FC<Props> = ({ label, value, className }) => {
       {value}
     </Td>
   );
+  const formattedValue = isJson(value)
```
Comment (Stijnkool) [inline src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx:35]: Would it not be simpler if you had a function `formatValue` that just returns this string? The `isJson` function can be a part of that. Just an idea. Not required.

**Review by Stijnkool — APPROVED** [review]
👍

File: src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx (hunk 1 above)
Comment (andraskvr) [reply src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx]: It was a good idea to extract that as well :+1:

File: src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx [hunk 2]
```diff
@@ -32,9 +32,7 @@ export const CellWithCopy: React.FC<Props> = ({ label, value, className }) => {
       {value}
     </Td>
   );
-  const formattedValue = isJson(value)
-    ? JSON.stringify(JSON.parse(value), null, 2)
-    : value;
+  const formattedValue = formatValue(value);
```
Comment (Stijnkool) [inline src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx:35]: You could even inline this function call. No need for a variable.

**Pull request closed without merging**